azure-pipelines.release-fluentui.yml

```diff
@@ -16,23 +22,31 @@ steps:
       versionSpec: '10.x'
     displayName: 'Install Node.js'
 
+  - script: |
+      ver=`node -p "require('./packages/fluentui/react-northstar/package.json').version"`
+      echo "##vso[task.setvariable variable=deployBasePath]$ver"
+      echo "##vso[task.setvariable variable=docsiteVersion]$ver"
+      echo "Docsite base path published to for version $ver"
+    displayName: Set base path and version variables for job
   - script: |
       git config user.name "Fluent UI Build"
       git config user.email "fluentui-internal@service.microsoft.com"
       git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/microsoft/fluentui.git
     displayName: Authenticate git for pushes
-
   - script: |
-      npx midgard-yarn install
+      yarn install
     displayName: yarn
-
   - script: |
       PRODUCTION=true yarn build
     displayName: yarn build
-
   - script: |
       NODE_ENV=production yarn build:fluentui:docs
     displayName: yarn build:fluentui:docs
+  - task: PublishBuildArtifacts@1
+    displayName: 'Publish docsite as pipeline artifact'
+    inputs:
+      PathtoPublish: 'packages/fluentui/docs/dist'
+      ArtifactName: 'docsite_v$(docsiteVersion)'
```

Comment:
Is it required?

Reply:
no, I think but it's handy to have the artifacts available... storage is cheap and we can backfill easily if we ever move storage or have to clean it for some reason